### DISCUSSION TITLE
Hey what do you do to get the sun looking like that?

### DISCUSSION THREAD

**Discussion by J-McFc** [zesterer/openmw-shaders#3] [General]
The shaders all work great, but I've still got the old shoddy sun lol, did you do anything to it?

**Comment by zesterer** [comment]
Check out my volumetric cloud & mist mod, which works with the most recent nightly builds: https://github.com/zesterer/openmw-volumetric-clouds

The mod includes some improvements to sun rendering.

Note that, until tomorrow, you'll need to change the `vec3 light_pos = ...` line in `shaders/clouds.omwfx` to `vec3 light_pos = (omw.invViewMatrix * vec4(omw_GetPointLightViewPos(i), 1.0)).xyz;` for the mod to work. Tomorrow's nightly build of OpenMW won't require this change though.
